tests/queries/test_slippage_investigation.py

```diff
@@ -125,6 +136,90 @@ def test_positive_slippage_evaluation(self):
         # https://dune.com/queries/1836718?CTE_NAME_e15077=final_token_balance_sheet&EndTime_d83555=2023-01-02+00%3A00%3A00&StartTime_d83555=2023-01-01+00%3A00%3A00&TxHash_t6c1ea=0x5c4e410ce5d741f60e06a8621c6f12839ad39273f5abf78d4bbc1cd3b31de46c
         self.assertEqual(tx_slippage["num_entries"], 4)
 
+    def test_slippage_regression(self):
+        """
+        Two known batches with ETH transfers previously not captured.
+        The following hashes were picked up by this query: https://dune.com/queries/1957339
+        which displays batches where these missing ETH transfers occurred.
+        Bug was picked up by our Dune Alerts on Feb. 2, 2023
+        https://cowservices.slack.com/archives/C03PW4CR38A/p1675328564060139
+        """
+        old_query_id = 1836718
+        # For TX: 0x3b2e9675b6d71a34e9b7f4abb4c9e80922be311076fcbb345d7da9d91a05e048
+        old_query_results = self.dune.get_result(job_id="01GRA17C0WFB27W0TC5TZ7BFP1")
```

Comment:
how long will this job_id be available?
I am okay with only calculating the new slippage values, but not verifying that it was an issue before

Reply:
2 year expiry.